Makefile

```diff
@@ -92,6 +92,11 @@ ifneq "$(LXD_OFFLINE)" ""
 	exit 1
 endif
 	go get -t -v -d -u ./...
+
+	# Manual fixups
+	go get github.com/mdlayher/ndp@v0.8.0
+	go get github.com/canonical/dqlite@HEAD
```

Comment:
I realise this is a cherry-pick but I'm a bit confused by it. As far as I can tell `github.com/canonical/dqlite` doesn't contain any Go code? Also, what does `@HEAD` do here?

Reply:
Indeed, that was not doing anything useful/effective so I dropped it. Thanks!